pandas/io/common.py

```diff
@@ -115,24 +115,8 @@ def _stringify_path(filepath_or_buffer):
     Any other object is passed through unchanged, which includes bytes,
     strings, buffers, or anything else that's not even path-like.
     """
-    try:
-        import pathlib
-        _PATHLIB_INSTALLED = True
```

Comment:
so in order to make this pass 3.5, I think you might need to:

```
import pathlib
...

if hasattr(filepath_or_buffer, '__fspath__'):
    ....
elif isintance(filepath_or_buffer, pathlib.Path):
   return str(filepath_or_buffer)
```

Reply:
Ah. I'll change this.